embeddings/pipeline/lightning_classification.py

```diff
@@ -22,6 +22,7 @@ class LightningClassificationPipeline(
     DEFAULT_TASK_MODEL_KWARGS = {"use_scheduler": True}
     DEFAULT_DATAMODULE_KWARGS = {"max_seq_length": None}
     DEFAULT_MODEL_CONFIG_KWARGS = {"classifier_dropout": None}
+    DEFAULT_EARLY_STOPPING_KWARGS = {"monitor": "val/Loss", "mode": "min", "patience": 1}
```

Comment:
Early stopping should be disabled if validation subset is not present (e.g. sequence labeling case)